config/routes.rb

```diff
@@ -97,6 +97,7 @@
       member do
         patch :set_normal_mode
         patch :set_independent_mode
+        post :leave
```

Comment:
Two questions:
1) Do we want to use `destroy` rather than making a new action
2) If "no" to 1, do we want to set this to `delete :leave` instead of `post :leave`

Reply:
I'm really unsure. I just feel that using a `delete` action has extra meaning that it deletes a record, and using a `post` is more neutral and can mean any action.